tests/test_loggers.py

```diff
@@ -95,6 +99,9 @@ def test_configure_logging__with_path_to_log_folder_and_file_name__uses_path_as_
         assert file_handler.baseFilename == os.path.join(
             tmp_dir, "my_log__2020_07_15_103508.txt"
         )
+        if is_system_windows():
+            # Tanner (8/7/20): windows raises error if file is not closed
+            file_handler.close()
```

Comment:
we might as well close the file handler on all systems right? not just windows? probably best practice